gazebo/rendering/LensFlare.hh

```diff
@@ -63,7 +124,20 @@ namespace gazebo
       public: void SetColor(const ignition::math::Vector3d &_color);
 
       /// \brief Update function to search light source
-      private: void Update();
+      protected: void Update();
+
+      /// \brief Set camera stored in LensFlarePrivate
+      /// \param[in] _camera Camera to use in sensor.
+      protected: void SetCameraSensor(CameraPtr _camera);
```

Comment:
note that `_camera` can be either a camera sensor or a user camera but this name suggests that it needs to be a camera sensor.

Maybe something like `SetCameraImp`?  We've used the the *Imp suffix in other places for internal functions.

Reply:
nice catch! updating to `LensFlare::SetCameraImpl` in #2975